src/Homework3/4_couple_elements.py

```diff
@@ -0,0 +1,16 @@
+"""Дан список чисел. Посчитайте, сколько в нем пар элементов,
+равных друг другу. Считается, что любые два элемента,
+равные друг другу образуют одну пару, которую необходимо посчитать.
+Входные данные - строка из чисел, разделенная пробелами.
+Выходные данные - количество пар.
+Важно: 1 1 1 - это 3 пары, 1 1 1 1 - это 6 пар.
+
+"""
+
+list1 = [1, 2, 1, 7, 1, 2, 1, 7, 1, 1, 3,
+         4, 3, 5, 3, 7, 3, 4, 4, 5, 4, 3,
+         ]
+for el in set(list1):
+    print('Для значения {el} количество пар в списке равно {result}'
+          .format(el=el,
+                  result=(list1.count(el) * (list1.count(el) - 1)) // 2))
```

Comment:
it runs count twice. it can be counted one time

Reply:
done